.nucleus.yaml

```diff
@@ -30,6 +30,9 @@ branches:
   spring22:
     pull-request:
       <<: *branch-definition
+  summer22:
+    pull-request:
+      <<: *branch-definition
```

Comment:
Ah, it worked because it's in `master`. Super confusing. 🤦 

https://github.com/salesforce/lwc/blob/5af18fdd904bc6cfcf7b76f3c539490ff11515b2/.nucleus.yaml#L29-L31